cmd/identity/batch.go

```diff
@@ -40,6 +40,8 @@ var (
 	}
 
 	keyCfg struct {
+		// TODO: where is this used and should it be conistent with "latest" alias?
+		VersionNumber uint   `default:"0" help:"version of identity (0 is latest)"`
```

Comment:
I agree that we should be consistent but I think it's better to have the default first because the help text is long and it makes it inconvenient to look up what the default is or if there even is one.